src/xcode/ENA/ENA/Source/Services/HealthCertificateValidation/HealthCertificateValidationService.swift

```diff
@@ -62,15 +64,34 @@ final class HealthCertificateValidationService: HealthCertificateValidationProvi
 	) {
 		
 		// 1. Apply technical validation
-		let expirationDate = healthCertificate.cborWebTokenHeader.expirationTime
 		
+		let signatureInvalid: Bool
+		var signatureValidationError: Error?
+		// DONT FORGET: pass singatures array into this function!
+		let result = signatureVerifying.verify(certificate: healthCertificate.base45, with: [], and: validationClock)
+		switch result {
+		case .success:
+			signatureInvalid = false
+		case.failure(let error):
+			signatureInvalid = true
+			signatureValidationError = error
+		}
+		
+		let expirationDate = healthCertificate.cborWebTokenHeader.expirationTime
 		// NOTE: We expect here a HealthCertificate, which was already json schema validated at its creation time. So the JsonSchemaCheck will here always be true. So we only have to check for the expirationTime of the certificate.
```

Comment:
This comment is outdated. And I guess it best should be moved right after `// 1. Apply technical validation` because it explains the whole step.

```suggestion
		// NOTE: We expect here a HealthCertificate, which was already json schema validated at its creation time. So the JsonSchemaCheck will here always be true. So we only have to check for the expirationTime and the signature validity of the certificate.
```

Reply:
Done